scripts/OMakefile

```diff
@@ -44,14 +44,19 @@ install:
 	$(IPROG) rio.db.conf.skel $(DESTDIR)$(ETCDIR)/db.conf.rio
 	mkdir -p $(DESTDIR)/etc/init.d
 	mkdir -p $(DESTDIR)/etc/rc.d/init.d
-	$(IPROG) init.d-xapi $(DESTDIR)/etc/rc.d/init.d/xapi
+	$(IPROG) init.d-xapi $(DESTDIR)$(LIBEXECDIR)/xapi-init
+	$(IPROG) init.d-xapi-old $(DESTDIR)/etc/init.d/xapi
 	$(IPROG) init.d-xapissl $(DESTDIR)$(LIBEXECDIR)/xapissl
-	$(IPROG) init.d-xapi-domains $(DESTDIR)/etc/rc.d/init.d/xapi-domains
-	$(IPROG) init.d-management-interface $(DESTDIR)/etc/rc.d/init.d/management-interface
-	$(IPROG) init.d-genptoken $(DESTDIR)/etc/rc.d/init.d/genptoken
-	$(IPROG) init.d-save-boot-info $(DESTDIR)/etc/rc.d/init.d/save-boot-info
-	$(IPROG) init.d-attach-static-vdis $(DESTDIR)/etc/rc.d/init.d/attach-static-vdis
-	$(IDATA) cdrommon@.service $(DESTDIR)/usr/lib/systemd/system/cdrommon@.service
+	$(IPROG) init.d-management-interface $(DESTDIR)$(LIBEXECDIR)/legacy-management-interface
+	$(IPROG) init.d-save-boot-info $(DESTDIR)$(LIBEXECDIR)/save-boot-info
+	$(IPROG) init.d-attach-static-vdis $(DESTDIR)$(LIBEXECDIR)/attach-static-vdis
+	$(IPROG) cdrommon@.service $(DESTDIR)/usr/lib/systemd/system/cdrommon@.service
```

Comment:
Yes, but I was talking about `cdrommon@.service` and the new `*.service` files below.